app/portainer/settings/general/index.js

```diff
@@ -2,5 +2,10 @@ import angular from 'angular';
 
 import { sslCertificate } from './ssl-certificate';
 import { openAMT } from './open-amt';
+import { FDO } from './fdo';
```

Comment:
just for consistency with the other I would change this for lower case
```suggestion
import { fdo } from './fdo';
```